hazelcast-documentation/src/SpringIntegration.md

```diff
@@ -228,6 +228,7 @@ Hazelcast Distributed `ExecutorService`, or more generally any Hazelcast managed
 
   <hz:hazelcast id="instance">
     <hz:config>
+      <hz:spring-aware />
```

Comment:
i am not sure about tradition but adding attribute `enabled` seems more sensible to me.
like `<hz:spring-aware enabled="true"/>` , 
but without attribute still ok for me.